plugins/available/basher.plugin.bash

```diff
@@ -0,0 +1,12 @@
+cite about-plugin
+about-plugin 'initializes basher, the shell package manager'
+
+# https://github.com/basherpm/basher
+
+if ! _command_exists basher && [[ -x "$HOME/.basher/bin/basher" ]]; then
+	pathmunge "$HOME/.basher/bin"
+else
+	return 0
+fi
+
+eval "$(basher init - bash)"
```

Comment:
@cornfeedhobo what do you say about adding a log at the end of files like `basher initialized successfuly`?  
maybe we can do that is `reloader.bash`, but this will require us to return 1 on failed loads, which is hard to do.
I think adding this kind of message to each file is very tedious, what are your thoughts on this?

Reply:
I'm always supportive of expanding logging

Reply:
@NoahGorny Yeah, [the way we source](https://github.com/Bash-it/bash-it/blob/master/scripts/reloader.bash#L28) silently swallows errors because we don't use `set -x` and a trap to catch errors.

In haste I've added a success message, but I would vote that this gets addressed in `reloader.bash` instead.

Reply:
Well... I started diving down the rabbit hole of patching up reloader to handle any issues while sourcing, but it's non-trivial. hmm....

Reply:
I guess we should remove the success message here, and open up an issue about improving the reloader